packages/react/src/Alert/Alert.test.tsx

```diff
@@ -0,0 +1,114 @@
+import { fireEvent, render } from '@testing-library/react'
+import { createRef } from 'react'
+import { Alert } from './Alert'
+import '@testing-library/jest-dom'
+
+describe('Alert', () => {
+  it('renders', () => {
+    const { container } = render(<Alert />)
+
+    const component = container.querySelector(':only-child')
+
+    expect(component).toBeInTheDocument()
+    expect(component).toBeVisible()
+  })
+
+  it('renders a design system BEM class name', () => {
+    const { container } = render(<Alert />)
+
+    const component = container.querySelector(':only-child')
+
+    expect(component).toHaveClass('amsterdam-alert')
+  })
+
+  it('can have a additional class name', () => {
+    const { container } = render(<Alert className="extra" />)
+
+    const component = container.querySelector(':only-child')
+
+    expect(component).toHaveClass('extra')
+
+    expect(component).toHaveClass('amsterdam-alert')
+  })
+
+  it('supports ForwardRef in React', () => {
+    const ref = createRef<HTMLDivElement>()
+
+    const { container } = render(<Alert ref={ref} />)
+
+    const component = container.querySelector(':only-child')
+
+    expect(ref.current).toBe(component)
+  })
+
+  it('renders a design system BEM class name', () => {
+    const { container } = render(<Alert />)
+
+    const component = container.querySelector(':only-child')
+
+    expect(component).toHaveClass('amsterdam-alert')
```

Comment:
Do we want to check here if `amsterdam-alert__content` is rendered as well?

Reply:
That div is actually always there.

Reply:
But we do want to check if it's there, no? If we remove it in future MRs, we're introducing a breaking change in the public facing interface.

Reply:
Let's leave this as is for now, and discuss our testing strategy with the other devs.